wrapper/src/main/java/software/amazon/jdbc/dialect/RdsMultiAzDbClusterMysqlDialect.java

```diff
@@ -47,7 +47,9 @@ public class RdsMultiAzDbClusterMysqlDialect extends MysqlDialect {
 
   private static final String FETCH_WRITER_NODE_QUERY_COLUMN_NAME = "Source_Server_Id";
 
-  private static final String NODE_ID_QUERY = "SELECT @@server_id";
+  private static final String NODE_ID_QUERY = "SELECT id, SUBSTRING_INDEX(endpoint, '.', 1)"
```

Comment:
Can we add a comment explaining the difference between what is returned by id and what is returned from the SUBSTRING call?